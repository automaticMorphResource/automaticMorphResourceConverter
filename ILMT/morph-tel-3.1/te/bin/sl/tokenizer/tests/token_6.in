భూమికి చాలా ఎత్తు నుంచి కూడా ఇది కనిపిస్తుందని చెబుతుంటారు. 
