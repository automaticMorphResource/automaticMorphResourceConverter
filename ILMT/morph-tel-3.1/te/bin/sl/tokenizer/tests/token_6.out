<Sentence id="1">
1	భూమికి	unk
2	చాలా	unk
3	ఎత్తు	unk
4	నుంచి	unk
5	కూడా	unk
6	ఇది	unk
7	కనిపిస్తుందని	unk
8	చెబుతుంటారు	unk
9	.	unk
</Sentence>

